Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java

```diff
@@ -0,0 +1,149 @@
+package mage.cards.a;
+
+import mage.MageInt;
+import mage.abilities.TriggeredAbility;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.continuous.GainAbilityControlledEffect;
+import mage.abilities.keyword.DoubleStrikeAbility;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.permanent.TokenPredicate;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.PermanentToken;
+import mage.game.permanent.token.BirdToken;
+import mage.watchers.Watcher;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * @author AsterAether
+ */
+public final class AkimTheSoaringWind extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("Creature tokens");
+
+    static {
+        filter.add(TokenPredicate.instance);
+    }
+
+    public AkimTheSoaringWind(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{U}{R}{W}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.BIRD);
+        this.subtype.add(SubType.DINOSAUR);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(4);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Whenever you create one or more tokens for the first time each turn, create a 1/1 white Bird creature token with flying.
+
+        this.addAbility(new AkimTheSoaringTokenAbility(), new AkimTheSoaringWindWatcher());
```

Comment:
If you add condition hint to card then you can improve card's usability for users. Search for example:
* `.addHint(new ConditionHint(LudevicNecroAlchemistCondition.instance, "Player other than you lost life this turn")));`

How to add:
1. Extract checkTrigger code to Condition object;
2. Use that condition in checkTrigger;
3. Add condition hint.

You can use same condition hints in your another PR (#6430).